scripts/formatting/format

```diff
@@ -125,6 +125,8 @@ all_prettier_files() {
     -o -path "./fsharp-backend/.paket" \
     -o -path "fsharp-backend/Build" \
     -o -path "./fsharp-backend/Build" \
+    -o -path "fsharp-backend/tests/staticassets-tests" \
+    -o -path "./fsharp-backend/tests/staticassets-tests" \
```

Comment:
is this to deal with OS inconsistencies?

Reply:
You mean the fact that it's there twice? I don't remember. I'm cargo culting it a bit here.

Reply:
yeah that's what I meant; OK, np